client.go

```diff
@@ -25,6 +34,77 @@ func Dial(network, address string) (*Client, error) {
 	return NewClient(conn)
 }
 
+// DialConfig is used to pass configuration to DialURI()
+type DialConfig struct {
+	DTLSConfig dtls.Config
+	TLSConfig  tls.Config
+
+	Net transport.Net
+}
+
+// DialURI connect to the STUN/TURN URI and then
+// initializes Client on that connection, returning error if any.
+func DialURI(uri *URI, cfg *DialConfig) (*Client, error) {
+	var conn Connection
+	var err error
+
+	nw := cfg.Net
```

Comment:
```suggestion
	var network cfg.Net
```

Reply:
`cfg.Net` isnt a type but a member of the `cfg` struct.

Reply:
oh hah!